server/src/utils/MixerConnection.ts

```diff
@@ -248,19 +261,23 @@ export class MixerGenericConnection {
         this.mixerConnection[0].updatePflState(channelIndex)
     }
 
-    updateMuteState = (faderIndex: number) => {
+    updateMuteState = (faderIndex: number, allMixersButThis: number = -1) => {

```

Comment:
Does `allMixersButThis` represent the mixer index that is the owner of the update?

Reply:
Looking at the implementation it's to specify what mixer **not** to update...